### PR TITLE
Global styles

### LINKED ISSUE #10: Global Styling
# User Story
As a **programmer**,
I want to **add global styling**,
so I can **have my basic design implemented**.

# Description

#Material
image

# Done

- [x] Add global styles file
- [x] Connect global styles to app.js
- [ ] Write global styles into file from design

### PR DESCRIPTION
**Pull request by LebkuchenCJ**
Fix #10 

As a programmer,
I want to add global styling,
so I can have my basic design implemented.

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -55,11 +55,19 @@ Header.propTypes = {
...
   justify-content: space-between;
   align-items: center;
+  text-align: center;
+
+  @media only screen and (max-width: 600px) {
+    h1 {
+      font-size: 1.8rem;
+      flex-grow: 2;
+    }
+  }
```
Comment (marwinburesch) [inline client/src/components/Header.js]: I don't see how this is part of global styles :P
🔬

File: client/src/index.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
```
Comment (marwinburesch) [inline client/src/index.js:3]: lovely

File: client/src/GlobalStyles.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        #root {
+          background-color: var(---bg-color-main);
+        }
+
+        body {
+          height: 100vh;
+          width: 100vw;
+          margin: 0;
+          padding: 0;
+          overflow: hidden;
```
Comment (marwinburesch) [inline client/src/GlobalStyles.js]: `height`, `width` and `overflow`: I'd put these on the watchlist of potential issues for later. Whenever there is weird scrollbars, fiddle around with these :)
Also don't expect `height: 100vh` to work perfectly on real mobile devices (not the dev toosl setting)

**Review by marwinburesch — APPROVED** [review]
Good stuff mate!
Just some things for future reference and a little nitpicky comment :)
🐜

File: client/src/components/Header.js (hunk 1 above)
Comment (LebkuchenCJ) [reply client/src/components/Header.js]: I know... but it affected my global viewing ;) I will try not to mix it up in future PR

**Pull request merged**